pkg/internal/common/flags/common.go

```diff
@@ -14,5 +14,6 @@ func GetSignerFlags() []cli.Flag {
 		&FireblocksTimeoutFlag,
 		&FireblocksSecretStorageTypeFlag,
 		&Web3SignerUrlFlag,
+		&SilentFlag,
```

Comment:
Don't add it here. It's explicitly says signer flags and silent is not a part of it.

Reply:
Removed